vsintegration/src/vs/FsPkgs/FSharp.LanguageService/FSharp.LanguageService.Base/CodeWindowManager.cs

```diff
@@ -14,7 +14,6 @@
 using Microsoft.VisualStudio.FSharp.LanguageService.Resources;
 
 namespace Microsoft.VisualStudio.FSharp.LanguageService {
-    /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager"]/*' />
```

Comment:
sure, as i said, easy to remove this commit from pr. 
With pr is easier to revert than ask and wait for permission 

Btw maybe if these file are needed, the files exists in an internal repo of doc team? Doc is a good candidate for open source contribution.
just curious of msdn page source like [List.append](https://msdn.microsoft.com/en-us/library/ee353428.aspx) with example etc :smile: